styles/globals.css

```diff
@@ -117,4 +117,11 @@
   .summary-link a {
     @apply text-default-text underline;
   }
+  .is-sticky {
```

Comment:
I am not sure why you made it wider when it starts floating, this causes overlap with the questions. See the dimensions in my screenshot to reproduce (or just resize and you should be able to see):
![image](https://user-images.githubusercontent.com/3630698/151221428-73297c61-01f6-45dd-aae8-4487cac29af3.png)

Reply:
It is also too far to the right when the page is wide. I think it should stay where it is, with the size it has, and just shift down, if possible.

Reply:
Keeping in the same spot will most likely cover up the same question you screen grabbed above, but I don't mind it too much as users can scroll up to see the questions. I can go ahead and try this change.

Reply:
Confirmed fixed!